CHANGELOG_PENDING.md

```diff
@@ -37,6 +37,7 @@ Friendly reminder, we have a [bug bounty program](https://hackerone.com/tendermi
   - [libs/os] Kill() and {Must,}{Read,Write}File() functions have been removed. (@alessio)
 
 - Blockchain Protocol
+  - [store/state/evidence/light] \#5771 Use an order-preserving varint key encoding (@cmwaters)
```

Comment:
Is this the right place for it? Seems like the most relevant section

Reply:
No, this isn't related to the blockchain protocol, this is only internal state storage. I'd add a separate section for it if nothing else is appropriate.